src/documents/IoCSS.md

```diff
@@ -140,3 +149,61 @@ fluid.defaults("fluid.moduleLayoutHandler", {
     }
 });
 ```
+
+## Example: `source` ##
+```javascript
+fluid.defaults("fluid.tests.uploader", {
+    gradeNames: ["fluid.component"],
+    components: {
+        uploaderContext: {
+            type: "fluid.progressiveCheckerForComponent",
+            options: {componentName: "fluid.tests.uploader"}
+        },
+        uploaderImpl: {
+            type: "fluid.tests.uploaderImpl"
+        }
+    },
+    distributeOptions: [{
+        target: "{that > uploaderImpl}.options" // Target a directly nested component matching the context "uploaderImpl"
+        source: "{that}.options",               // Distribute ALL of our options there, except exclusions:
+        exclusions: ["components.uploaderContext", "components.uploaderImpl"], // options targetted directly at these subcomponents are left undisturbed in place
+
+    }],
+    progressiveCheckerOptions: {
+        checks: [{
+            feature: "{fluid.test}",
+            contextName: "fluid.uploader.html5"
+        }]
+    }
+});
+```
+
+## Example: broadcast via root to all components ##
+
+([All Husnock, everywhere](http://en.memory-alpha.org/wiki/Husnock)) 
+
+```javascript
+
+// A `mixin grade` to contribute two listeners to every viewComponent in the system
+fluid.defaults("fluid.debug.listeningView", {
+    listeners: {
+        onCreate: {
+            funcName: "fluid.debug.viewMapper.registerView",
+            args: ["{fluid.debug.viewMapper}", "{that}", "add"]
+        },
+        onDestroy: {
+            funcName: "fluid.debug.viewMapper.registerView",
+            args: ["{fluid.debug.viewMapper}", "{that}", "remove"]
+        }
+    }
+});
+
+fluid.defaults("fluid.debug.viewMapper", {
+    gradeNames: ["fluid.component", "fluid.resolveRoot"],
+    // Distribute the grade `fluid.debug.listeningView` to every viewComponent in the system
+    distributeOptions: {
+        record: "fluid.debug.listeningView",
+        target: "{/ fluid.viewComponent}.options.gradeNames" // <- use of IoCSS root context "/"
+    }
+});
+```
```

Comment:
The variety and simplicity of all these examples is great. Bonus points for sneaking in the Star Trek reference. I love that episode of TNG.